src/components/input/elastic-textarea.cypress.ts

```diff
@@ -0,0 +1,67 @@
+import textInputHTML from './input.twig';
+import { createElasticTextArea } from './elastic-textarea';
+
+import { render } from '../../../cypress/cypress-render-component';
+
+describe('Elastic Textarea', () => {
+  it('Resizes correctly with no rows attribute', () => {
+    render(
+      textInputHTML({
+        class: 'c-input--elastic js-elastic-textarea',
+        type: 'textarea',
+      })
+    );
+    const textarea = cy.get('.js-elastic-textarea');
+
+    textarea.invoke('css', 'max-width', '500px');
```

Comment:
What's the `max-width` for?

Reply:
Oh wait, I see — it's to make a predictable width so you know when your sample text will wrap?

Reply:
Yes. Before, it was full-screen-width (which is known and can be configured in cypress) but that would mean that I would need to put more words in my sentence in order for it to wrap 😂